Engineering/Mobile/Kotlin coroutine.md

```diff
@@ -0,0 +1,170 @@
+---
+tags: Mobile, Kotlin
```

Comment:
Maybe it's useful to add `android` tag? Also, tags should be in lowercase.

```suggestion
tags: mobile, kotlin, android
```

Reply:
Actually, this one does not have anything related to android, pure Kotlin feature 🤔